proto/opamp.proto

```diff
@@ -48,7 +48,7 @@ message AgentToServer {
 
     // The current health of the Agent.
     // May be omitted if nothing changed since last AgentToServer message.
-    AgentHealth health = 5;
+    ComponentHealth health = 5;
```

Comment:
Do we need to say that the top-level component represents the Agent as a whole?

Reply:
I added a clarification. Let me know if it needs anything further.